doc/api/util.md

```diff
@@ -1020,6 +1020,81 @@ Otherwise, returns `false`.
 See [`assert.deepStrictEqual()`][] for more information about deep strict
 equality.
 
+## `util.parseArgs([config])`
```

Comment:
Matteo is against that: https://github.com/nodejs/node/pull/42675#issuecomment-1094124517 https://github.com/nodejs/node/pull/42675#issuecomment-1094287657. We can wait for him to get back from vacation to discuss that, or leave it in `node:util`.

Reply:
> process is already overloaded with so much functionality

Hmm, did we ever have any max limit on the number of functionalities we can expose from an API? If it's a closely related functionality (I think it is in this case), I think it makes sense to place it there.

Reply:
> Since we don't have any time pressure now

I would still like to have this landed soon, as it's a burden to keep updating a PR like this as more reviews float in.

Reply:
I don't care either way, if @mcollina can be convinced to go with `process` rather than `util`.

Reply:
I'm -1 on adding anything non-essentially to a global object. An utility module falls into this category.

I'm ok with _anything_ but `process`.

Reply:
Why, though? What's wrong with adding it to the global object?

Reply:
A few attacks I've seen use functions that are attached to process because process is available in scope.

Reply:
so is `require`, though.

Reply:
I agree! Let's not add more surface area in critical places.

Reply:
I've added the @nodejs/tsc agenda tag, specifically hoping we can address this point of contention.

Reply:
Apart from the security aspects which we still need some clarification on, another point I was thinking about is - do we imagine parseArgs to effectively replace raw uses of process.argv? If we plan to make this the primary way of handling cli args, imo it makes even more sense to put this on process than util.

Reply:
Yes, I would absolutely hope and expect that to be the case.